torchmetrics/metric.py

```diff
@@ -330,19 +330,19 @@ def wrapped_func(*args, **kwargs):
         return wrapped_func
 
     @abstractmethod
-    def update(self) -> None:  # pylint: disable=E0202
+    def update(self, *_: Any, **__: Any) -> None:  # pylint: disable=E0202
         """
         Override this method to update the state variables of your metric class.
         """
 
     @abstractmethod
-    def compute(self):  # pylint: disable=E0202
+    def compute(self) -> Any:  # pylint: disable=E0202
```

Comment:
IIRC TorchScript will not understand that now, but probably the right thing here is to define a type parameter T and have Metric[T] and then `compute() -> T`

Reply:
ok, i ll drop it for now, but mind send a PR?